src/main/java/de/rabea/game/Game.java

```diff
@@ -26,12 +26,15 @@ public void play(Board board) {
 
     private void switchPlayer(Board board) {
         if (gameIsNotOver(board)) {
-            currentPlayer = currentPlayer == player1 ? player2 : player1;
+            if (currentPlayer == player1) {
+                currentPlayer = player2;
+            }
+            else currentPlayer = player1;
         }
     }
 
     private Board playOneRound(Player player, Board board) {
-        return board.placeMark(player.makeMove(board), player.mark());
+        return board.placeMark(player.getMove(board), player.mark());
     }
```

Comment:
Jim's suggestion was that I shouldn't reach into the player twice here. And he said it seems strange to pass the board into `makeMove()` inside a method that is called on the board. I'm not sure how best to rectify that.